tests/attr/layer/test_layer_integrated_gradients.py

```diff
@@ -126,7 +130,12 @@ def _assert_compare_with_emb_patching(self, input, baseline, additional_args):
         assertArraysAlmostEqual(delta, delta_with_ig)
 
     def _assert_compare_with_expected(
-        self, model, target_layer, test_input, expected_ig, additional_input=None,
+        self,
+        model: Module,
+        target_layer: Module,
+        test_input: Union[Tensor, Tuple[Tensor, ...]],
+        expected_ig: Tuple[List[List[float]], List[List[float]]],
```

Comment:
nit: This is correct for the existing usage, but to make this more generic, could change this to Tuple[List[List[float]], ...], so we can also pass arbitrary length tuples in the future.

Reply:
Agreed, done.